src/main/java/com/maxrt/shoeshop/shoes/FlipFlops.java

```diff
@@ -0,0 +1,10 @@
+package com.maxrt.shoeshop.shoes;
+
+import lombok.ToString;
+
+@ToString(callSuper = true, includeFieldNames = true)
+public class FlipFlops extends Shoes {
+    public FlipFlops(final int size, final String manufacturer) {
+        super(ShoeType.SUMMER, size, manufacturer);
```

Comment:
this class has no extra fields comparing to the parent class - either add extra information to it or remove from the codebase

Reply:
Added extra fields to child classes